b2file.txt

```diff
@@ -1 +1,2 @@
 im in branch2
+im in branch 5
```

Comment:
this code might create performance issue

Reply:
get lost

Reply:
where